contracts/oracles/BinanceOracle.sol

```diff
@@ -82,6 +93,7 @@ contract BinanceOracle is AccessControlledV8, OracleInterface {
         if (asset == BNB_ADDR) {
             symbol = "BNB";
```

Comment:
Given we are setting `asset = WBNB`, we could set the symbol also to `WBNB`, taking it into account in the config. Wdty?